### DISCUSSION TITLE
Protection of master branch and merge rules

### DISCUSSION THREAD

**Discussion by catkira** [catkira/py3gpp#13] [Announcements]
I have protected the master branch against force pushed and configured it to only allow merges that preserve a linear history.

**Comment by catkira** [comment]
actually I think with these rules its ok to PR directly into master and not into dev branch first. What do you think @vborchsh ?

**Reply by vborchsh** [reply]
Fine for me. We can use tags for releases.

**Reply by catkira** [reply]
ok I will delete the dev branch then
